### ISSUE TITLE
Channel index determines causal direction

### ISSUE THREAD

**Issue by eunseokyang** [adityapt/deepcausalmmm#8]
Hello, thanks for this awesome library.

I noticed something that might confuse users though. The DAG learning uses an upper triangular mask, which means **the order of channels in your dataframe actually determines which causal relationships are even possible to learn**.

```python
# unified_model.py:185-190
# This enforces that channel `i` can only influence channel `j` if `i < j`
mask = torch.triu(torch.ones(n_media, n_media), diagonal=1)
self.register_buffer('tri_mask', mask) .
```

What do you think?

**Comment by eunseokyang** [comment]
Thanks!
